src/components/InformationPanel/InformationPanel.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { FC, useState } from 'react'
+import styled, { css } from 'styled-components'
+
+import { useTheme, Theme } from '../../hooks/useTheme'
+
+import { DialogBase } from '../Base'
+import { Icon, iconMap } from '../Icon'
+import { Heading, HeadingTagTypes } from '../Heading'
+import { SecondaryButton } from '../Button'
+
+type Props = {
+  children?: React.ReactNode
+  title: string
+  titleTag?: HeadingTagTypes
+  type?: 'success' | 'info' | 'warning' | 'error' | ''
+  className?: string
+  active?: boolean
+  controllable?: boolean
+  onClickOpener?: () => void
+  onClickCloser?: () => void
+}
+
+export const InformationPanel: FC<Props> = ({
+  children,
+  title,
+  titleTag = 'span',
+  type = 'info',
+  className,
+  active = true,
+  controllable = false,
+  onClickOpener,
+  onClickCloser,
+}) => {
```

Comment:
[MUST]
It is an indication about four props of `active`, `controllabe`, `onClickOpener`, `onClickCloser`.

First, controllable is not necessary. Because `InfomationPanel` is a simple component, you should always be able to control it from outside or inside the component.

Second, there is no need to separate `onClickOpener` and `onClickCloser`. All you need is `onClickTrigger`. You can pass the current `active` as an argument to the `onClickTrigger` handler.

Partially shows the implementation of these amendments.
Please try to reference 🙏 

```tsx
type Props = {
  title: string
  titleTag?: HeadingTagTypes
  type?: 'success' | 'info' | 'warning' | 'error' | ''
  openMessage?: string
  closeMessage?: string
  active?: boolean
  className?: string
  children: React.ReactNode
  onClickTrigger?: (active: boolean) => void
}

export const InformationPanel: FC<Props> = ({
  title,
  titleTag = 'span',
  type = 'info',
  openMessage = '開く',
  closeMessage = '閉じる',
  active: activeProps = true,
  className = '',
  children,
  onClickTrigger,
}) => {
  const theme = useTheme()

  let iconName: keyof typeof iconMap = 'fa-info-circle'
  let iconColor = theme.palette.TEXT_GREY

  switch (type) {
    case 'success':
      iconName = 'fa-check-circle'
      iconColor = theme.palette.MAIN
      break
    case 'info':
      iconName = 'fa-info-circle'
      iconColor = theme.palette.TEXT_GREY
      break
    case 'warning':
      iconName = 'fa-exclamation-triangle'
      iconColor = theme.palette.WARNING
      break
    case 'error':
      iconName = 'fa-exclamation-circle'
      iconColor = theme.palette.DANGER
  }

  const [active, setActive] = useState(activeProps)

  const handleClickTrigger = () => {
    if (onClickTrigger) {
      onClickTrigger(active)
    } else {
      setActive(!active)
    }
  }

  useEffect(() => {
    setActive(activeProps)
  }, [activeProps])

  return (
    <Wrapper className={className} themes={theme}>
      <Title themes={theme}>
        <TitleIcon name={iconName} color={iconColor} themes={theme}></TitleIcon>
        <Heading type="blockTitle" tag={titleTag}>
          {title}
        </Heading>
        <PanelButton
          suffix={<Icon size={14} name={active ? 'fa-caret-up' : 'fa-caret-down'} />}
          size="s"
          onClick={handleClickTrigger}
        >
          {active ? closeMessage : openMessage}
        </PanelButton>
      </Title>
      {active && <Content themes={theme}>{children}</Content>}
    </Wrapper>
  )
}
```